docs/SEMANTIC_CONVENTIONS.md

```diff
@@ -0,0 +1,127 @@
+# Semantic Conventions
+
+While most of the core Boudicca Software is very generic (see our [Data Model](DATA_MODEL.md)),
+EventCollectors and most Publishers are specialized to work with Events,
+and this document describes how Events should look like for them to be interoperable.
+
+### Data types
+
+We define some custom data types we need our properties to be in. The values in () will denote the
+underlying [Format Variants](DATA_MODEL.md#format-variant) which are applicable for this data type, ordered by their
+priority.
+Earlier Format Variants are selected over later ones.
+
+* `name (format=text)`: datatype for the event name, since that one has to be text
+* `text (format=markdown,text)`: a text or markdown for structured text, to describe some property
+* `date (format=date)`: [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) Timestamp as text, for
+  example: `2009-06-30T18:30:00+02:00`
+* `number (format=number)`: A number, for example `1`,`0.5`,`-2`
+* `url (format=text)`: A URL as text
+* `list (format=list)`: A list of text elements
+* `enum<?> (format=text)`: Has to be one of the specified distinct values
+* `boolean (format=text)`: The text "true" or "false"
+
+Note: publishers have to be able to handle invalid values by for example simply ignoring them
+
+### Property List
+
+**Events have only two required properties: `name` and `startDate`**
+
+| PropertyName                                   | Meaning                                                                                                                         | [Data Type](#data-types) | [Usage](#usage-explanation) |
```

Comment:
how do you plan to keep this list up-2-date?

Reply:
by updating it when we change or introduce new properties :P   not sure what kind of strategy you want here?

Reply:
just ooc, manual updates usually run out of sync with code,. if it works, it works

Reply:
yes it does run out of sync, but i really cannot think of a better way right now.